css-cascade-7/Overview.bs

```diff
@@ -0,0 +1,319 @@
+		<li>Any of the above may be subdivided into named stylesheets with the ''@sheet'' keyword.
+	</ol>
+
+	A <dfn>sheet-name</dfn> is a string identifying
+	 inner named sheets within a stylesheet.
+
+	Syntactically, a stylesheet may include an explicit [=sheet-name=], and is represented
+	by the <<sheet-name>> in ''@sheet'' and ''@import'' rules,
+	which is a single <<ident>> token:
+
+	<pre class='prod'>
+		<dfn><<sheet-name>></dfn> = <<ident>>
+	</pre>
+
+<h3 id="at-ruledef-sheet">
+Named Stylesheets: the `@sheet` block at-rule</h3>
+
+	The ''@sheet'' [=block at-rule=]
+	assigns its child style rules to a particular named [=stylesheet=].
+	This sheet-assignment syntax is:
+
+	<pre class='prod'>
+		@sheet <<sheet-name>> {
+			<<rule-list>>
+		}
+	</pre>
+
+	Such ''@sheet'' block rules have the same restrictions and processing
+	as a [=conditional group rule=] [[CSS-CONDITIONAL-3]]
+	with a true condition.
+
+	<div class=example>
+		For example, ''@sheet'' and ''@media'' can be mixed:
+
+		<pre class=lang-css>
+		@sheet framework {
+			h1, h2 { color: maroon; background: white;}
+
+			@media (prefers-color-scheme: dark) {
+				h1, h2 { color: red; background: black; }
+			}
+		}
+		</pre>
+	</div>
+
+	Note: ''@sheet'' [=block at-rules=] cannot be interleaved with ''@import'' rules.
+
+<h3 id="sheet-declaration">
+Declaring Named Stylesheets</h3>
+
+	Named stylesheets can be declared:
+
+	* using an ''@import'' rule with the ''from'' keyword
+		assigning the contents of the imported file into that sheet.
+	* using an [[#at-ruledef-sheet|@sheet block at-rule]],
+		assigning its child style rules into that sheet.
+
+<h3 id="at-ruledef-sheet-block">
+Subdividing Stylesheets: the `@sheet` block at-rule</h3>
+
+<pre class='prod'>
+		@import [ <<string>> from ]?
+				[ <<url>> | <<string>> ]				
+		        [ layer | layer(<<layer-name>>) ]?
+		        <<import-conditions>> ;
+
+		<dfn export>&lt;import-conditions></dfn> = [ supports( [ <<supports-condition>> | <<declaration>> ] ) ]?
+		                     <<media-query-list>>?</pre>
+
+	where:
+	* <<string>> ''from'' imports only the named sheet
+		with name <<string>> from the provided URL.
+
+	* the <<url>> or <<string>>
+		gives the URL of the style sheet to be imported.
+
+	* the optional ''layer'' keyword or ''layer()'' function
+		assigns the contents of the style sheet
+		into its own anonymous [=cascade layer=]
+		or into the named [=cascade layer=].
+
+		The layer is added to the layer order
+		even if the import fails to load the stylesheet,
+		but is subject to any [=import conditions=]
+		(just as if declared by an ''@layer'' rule wrapped
+		in the appropriate [=conditional group rules=]).
+
+	* the optional <<import-conditions>>
+		states the [=import conditions=] under which it applies.
+
+	<div class="example">
+		The following <a href="#conditional-import">conditional <css>@import</css> rule</a>
+		only loads the style sheet with a name of ``foo`` from ``narrow.css`` when the UA
```

Comment:
foo and narrow.css aren't showing up as code in https://kurtcattischmidt.github.io/kurtspublishedw3cdrafts/AtSheet.html#example-00f8cdb8

Reply:
Same with the other example

Reply:
Should be good now